Documentation/contributing/development/contributing_guide.rst

```diff
@@ -14,29 +14,31 @@ Clone and Provision Environment
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 #. Make sure you have a `GitHub account <https://github.com/join>`_
-#. Clone the cilium repository into your ``GOPATH``.
+#. Fork the Cilium repository to your GitHub user or organization.
+#. Clone your ``${YOUR_GITHUB_USERNAME_OR_ORG}/cilium`` fork into your ``GOPATH``, and setup the base repository as ``upstream`` remote:
 
    ::
 
-      mkdir -p $GOPATH/src/github.com/cilium
-      cd $GOPATH/src/github.com/cilium
-      git clone https://github.com/cilium/cilium.git
+      mkdir -p ${GOPATH}/src/github.com/cilium
+      cd ${GOPATH}/src/github.com/cilium
+      git clone https://github.com/${YOUR_GITHUB_USERNAME_OR_ORG}/cilium.git
```

Comment:
After discussion with Quentin offline, let's just change to SSH for the origin. Using SSH for upstream could be a pain for external contributors.
```suggestion
      git clone git@github.com:${YOUR_GITHUB_USERNAME_OR_ORG}/cilium.git
```

Reply:
I read the discussion above, and I personally use SSH in all cases anyway but... why do we want to recommend usage of SSH for `origin`? It was HTTPS beforehand and I think it's perfectly fine, especially for beginners: they don't need to setup anything, the GH authentication takes care of the complexity for them 🤔

Reply:
It's annoying when you copy paste without thinking and you have double factor authentication. Probably fine if we assume beginners don't have double factor auth. enabled.

Reply:
Yes, that was also my assumption. I can add a note "if you have 2FA enabled..." for other folks though :p

Reply:
No, leave it. Let's not make it convoluted to please that one reviewer who's a pain :smile: